docs/source/concepts/CrystalField.rst

```diff
@@ -165,6 +165,30 @@ The difference between the two are discussed in more detail in the appendix__.
 __ `Appendix A: Wybourne Normalisation`_
 
 
+External and Molecular Magnetic Field Contributions
+---------------------------------------------------
+
+In addition to the crystal field Hamiltonian, there are two additional terms that contribute to the total Hamiltonian, the Hamiltonians for the external and molecular magnetic fields:
+
+.. math::
+  & H = H_{\mathrm{CEF}} + H_{\mathrm{EXT}} + H_{\mathrm{MOL}} \\
+
+The external field Hamiltonian describes the splitting of electronic energy levels of a magnetic ion in solids due to the presence of a magnetic field.
+The molecular field Hamiltonian describes the "internal" fields generated by the ordered moments of neighboring ions.
+By convention, the external magnetic fields are separated from the 'internal' magnetic field, which is also called a "molecular" field in older literature.
+Both of these effects can be accounted for in Mantid's calculation.
+The external applied field is known by the user, whilst the internal "molecular" field is (usually) unknown and needs to be fitted to measured data.
+
+In Mantid we define these two fields as:
+
+.. math::
+  & H_{\mathrm{EXT}} = g_J \mu_B \mathbf{J}\cdot\mathbf{B}_{\mathrm{EXT}} \\
+  & H_{\mathrm{MOL}} = 2 (g_J - 1) \mu_B \mathbf{J}\cdot\mathbf{B}_{\mathrm{MOL}}
+
+where :math:`g_J` is the Lande g-factor, :math:`\mu_B` is the Bohr magneton, :math:`\mathbf{B}` is the magnetic field vector (in Tesla in Mantid) and :math:`\mathbf{J}` is the total angular momentum operator vector.
+This definition follows the `FOCUS manual <https://epubs.stfc.ac.uk/manifestation/5723/RAL-TR-95-023.pdf>`_, found in reference `[Buckmaster72]`_.
```

Comment:
The `[Buckmaster72]` reference is incorrect. The FOCUS manual is not in the [references](https://github.com/mantidproject/mantid/blob/9b3689d8832b8b1cd6afe77b3544cec4a4e9ad9a/docs/source/concepts/CrystalField.rst#references) at all. I would just remove the text after the comma.

Reply:
I see! Yes I misinterpreted this from a mention later down the text. I have now removed it.